### DISCUSSION TITLE
Point Data Dir to folder with existing files?

### DISCUSSION THREAD

**Discussion by Shurov** [nextcloud/all-in-one#2883] [Questions] [answered]
Hi, I have clean AIO install. I've changed Data Dir as an env variable in docker compose file from the very beginning - prior to the first NC run. As well as doing `chown` and `chmod`. My Data Dir had already files there, so I hoped that NC will display them on the home screen. However, after launching NC - I couldn't see any files in Data Dir...
I was trying to debug for a long time - but everything seems correct and according to the official instructions. I tried to upload a new file into NC web interface - and it appeared in my targeted folder. So that the folder contains all old files + a newly uploaded one. While NC web interface shows only the latter. 
Since new file appeared in the correct folder, I understand that mapping is correct and working. From the terminal I can see that the new file has exactly the same owner and ownership rights as other previously existing files, so those are not different anyhow.
Thus my thought was (and a question) - is Nextcloud only able to see new files and can't see old existing files (prior to it's installation) in the given folder?

**Comment by szaimen** [comment, marked as answer]
Hi, as this is a general Nextcloud question (and not specifically about AIO), please ask your question at https://help.nextcloud.com. Thanks!
